Sources/NIOCore/BSDSocketAPI.swift

```diff
@@ -236,6 +236,11 @@ extension NIOBSDSocket.OptionLevel {
                 NIOBSDSocket.OptionLevel(rawValue: IPPROTO_TCP)
     #endif
 
+    /// Socket options that apply to MPTCP sockets.
+    ///
+    /// These only work on Linux currently.
+    public static let mptcp = NIOBSDSocket.OptionLevel(rawValue: 284)
```

Comment:
Does it make sense to mark them as unavailable from Darwin OSes?

Reply:
I'm inclined to leave them: it adds compiler errors for no good reason. We could set it to a nonsense value, I suppose.